song-server/src/main/java/org/icgc/dcc/song/server/repository/UploadRepository.java

```diff
@@ -24,8 +24,14 @@
 import org.skife.jdbi.v2.sqlobject.SqlQuery;
 import org.skife.jdbi.v2.sqlobject.SqlUpdate;
 import org.skife.jdbi.v2.sqlobject.customizers.RegisterMapper;
+import org.skife.jdbi.v2.sqlobject.stringtemplate.UseStringTemplate3StatementLocator;
+import org.skife.jdbi.v2.unstable.BindIn;
```

Comment:
removed this. Here is a message regarding BindIn from the jdbi creator himself
https://skife.org/jdbi/java/2011/12/21/jdbi_in_clauses.html